README.md

```diff
@@ -728,7 +728,7 @@ Document definitions are also modular. By invoking the `importDocumentDefinition
     }
 ```
 
-* `my-doc-type2.js`:
+* `my-doc-type2-fragment.js`:
```

Comment:
This now matches the name by which the file is imported in the example later on.